matminer/featurizers/tests/test_composition.py

```diff
@@ -101,6 +102,16 @@ def test_fere_corr(self):
         self.assertAlmostEqual(df_fere_corr["mean FERE correction"][0], 0.077146274)
         self.assertAlmostEqual(df_fere_corr["std_dev FERE correction"][0], 0.270209766)
 
+    def test_atomic_orbitals(self):
+        df_atomic_orbitals = AtomicOrbitals().featurize_dataframe(self.df, col_id="composition")
+        self.assertEqual(df_atomic_orbitals['HOMO_character'][0], 'd')
+        self.assertEqual(df_atomic_orbitals['HOMO_element'][0], Element('Fe'))
```

Comment:
don't return Element objects in the featurizer. Featurizers should either return floats or strings. The various scikit-learn machine learning algorithms tend to operate on floats or strings, but none of them can operate on an Element object